ossm-release-notes/ossm-release-notes-support-tables-assembly.adoc

```diff
@@ -0,0 +1,56 @@
+:_mod-docs_content-type: ASSEMBLY
+[id="ossm-release-notes-support-tables-assembly"]
+= {SMProductShortName} {SMProductVersion} feature support tables
+include::_attributes/common-attributes.adoc[]
+:context: ossm-release-notes-support-tables-assembly
+
+toc::[]
+
+These tables provide guidance on supported versions and feature availability in {SMProduct} 3.
+
+// Copied from OCP core file: making-open-source-more-inclusive.adoc.
+
+include::modules/ossm-release-notes-making-open-source-inclusive.adoc[leveloffset=+1]
+
+[id="release-notes-definitions"]
+== Definitions
+
+For {SMProductName} 3, features that are Generally Available (GA) are fully supported and are suitable for production use.
+
+Technology Preview (TP) features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. See the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal] for more information about TP features.
+
+Developer Preview (DP) features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
+
+Not available (NA) features may not be available with {SMProductName} 3.
+
+include::modules/ossm-release-notes-supported-versions.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources-supported-versions_{context}"]
+.Additional resources
+
+* For the latest supported versions of {Kialiproduct}, {KialiServer}, and {SMPlugin}, see: xref:../observability/kiali/ossm-console-plugin.adoc#ossm-install-console-plugin_ossm-console-plugin[Installing {SMPlugin}]
```

Comment:
Additional resources really shouldn't be a sentence, but if it is, it should be a proper one (add period and remove colon)

```suggestion
* For the latest supported versions of {Kialiproduct}, {KialiServer}, and {SMPlugin}, see xref:../observability/kiali/ossm-console-plugin.adoc#ossm-install-console-plugin_ossm-console-plugin[Installing {SMPlugin}].
```